main.go

```diff
@@ -100,6 +97,21 @@ func main() {
 
 	logger.Infof("rootConf: %+v", c)
 
+	// ********************************************************************************
+	// Configure Open Telemetry
+	// ********************************************************************************
+	if opentelemetry.IsEnabled() {
+		collectorAddress := c.OpenTelemetryEndpoint
+		spanExporter := opentelemetry.InitSpanExporter(ctx, collectorAddress)
+		metricExporter := opentelemetry.InitMetricExporter(ctx, collectorAddress)
+		o := opentelemetry.Init(ctx, spanExporter, metricExporter, "nsc")
+		defer func() {
+			if err = o.Close(); err != nil {
+				logger.Fatal(err)
```

Comment:
Please don't use fatal on deferring it can produce unexpected problems with other defers. Please have a look at https://go.dev/play/p/0LvgX3B1bn7